apps/js_urls/constants/regexp.py

```diff
@@ -0,0 +1,9 @@
+# Python Standard Library Imports
+import re
+
+
+URL_ARG_RE = re.compile(r"(\(.*?\))")
```

Comment:
These patterns are kind of convoluted.

Reply:
These are catching regex match groups in URLs like `(\d+)` and named match groups like `(?P<...>...)`